### PR TITLE
Can create or load script on Attach script dialog

### LINKED ISSUE #7016: 2 script icons on Scene dock
**Operating system or device - Godot version:**
compiled 681575fa7123592897090c6cce44402c4e45baeb

**Issue description** (what happened, and what was expected):
image
One for "create script" and the other one for "load script"
These 2 icons are not recognizable and take too much space.

It would be nice make it 1 icon in somehow.

### PR DESCRIPTION
**Pull request by volzhs**
Fix #7016, #6878 and it supersedes #6883

attach_script

### CODE REVIEW

**Review by bojidar-bg — APPROVED** [review]
Code looks good when read on a diagonal of 25 degrees... :smile: (i.e. don't assume I read it fully)

_Edit_: Can confirm it works :tada:

File: tools/editor/scene_tree_dock.cpp [hunk 1]
```diff
@@ -1976,20 +1933,13 @@ SceneTreeDock::SceneTreeDock(EditorNode *p_editor,Node *p_scene_root,EditorSelec
 
 
 	tb = memnew( ToolButton );
-	tb->connect("pressed",this,"_tool_selected",make_binds(TOOL_CREATE_SCRIPT, false));
-	tb->set_tooltip(TTR("Create a new script for the selected node."));
-	tb->set_shortcut(ED_GET_SHORTCUT("scene_tree/add_script"));
+	tb->connect("pressed",this,"_tool_selected",make_binds(TOOL_ATTACH_SCRIPT, false));
+	tb->set_tooltip(TTR("Create a new or load script for the selected node."));
```
Comment (volzhs) [inline tools/editor/scene_tree_dock.cpp]: I will change it right a way.

Comment (volzhs) [inline tools/editor/scene_tree_dock.cpp]: Done ! :)

**Review by vnen — APPROVED** [review]

**Comment by volzhs** [comment]
forgot to remove unused variable `ToolButton *button_load_script;` in `/tools/editor/scene_tree_dock.h`.
it's removed now.

**Pull request merged**